legal-api/src/legal_api/services/filings/validations/validation.py

```diff
@@ -12,10 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Common validation entry point for all filing submissions."""
+from http import HTTPStatus
 from typing import Dict
 
+from flask_babel import _
```

Comment:
We've been casting these to the name babel rather than _, 
where there are _ in the code, those are misses in reviews and are tech debt to clean up.
```suggestion
from flask_babel import _ as babel  # noqa: N813
```

Reply:
casted to `babel`